app/src/components/Market/OrderCard/components/OrderOptions/OrderOptions.tsx

```diff
@@ -93,76 +97,118 @@ const OrderOptions: React.FC = () => {
   return (
     <>
       <Box row alignItems="flex-start" justifyContent="center">
-        <StyledColumn>
-          <Box row justifyContent="center" alignItems="center">
-            <Label text={'Long Tokens'} />
-            <Spacer />
-            <StyledBalance>
-              {positions.loading ? (
-                <Loader size="sm" />
-              ) : !option.long ? (
-                '0.00'
-              ) : (
-                formatEtherBalance(option.long)
-              )}
-            </StyledBalance>
-          </Box>
-          <Button full size="sm" onClick={() => change(Operation.LONG)}>
-            Open Long
-          </Button>
-          <Spacer size="sm" />
-          <Button
-            full
-            disabled={!positions.loading ? false : true}
-            size="sm"
-            variant="secondary"
-            onClick={() => change(Operation.CLOSE_LONG)}
-          >
-            Close Long
-          </Button>
-        </StyledColumn>
-
-        <StyledColumn>
-          <Box row justifyContent="center" alignItems="center">
-            <Label text={'Short Tokens'} />
-            <Spacer />
-            <StyledBalance>
-              {positions.loading ? (
-                <Loader size="sm" />
-              ) : !option.short ? (
-                '0.00'
-              ) : (
-                formatEtherBalance(option.short)
-              )}
-            </StyledBalance>
+        <StyledTabs selectedIndex={tab} onSelect={(index) => setTab(index)}>
+          <Box row justifyContent="flex-start">
+            <StyledTabList>
+              <StyledTab active={tab === 0}>Long</StyledTab>
+              <Spacer />
+              <StyledTab active={tab === 1}>Short</StyledTab>
+              <Spacer />
+              <StyledTab active={tab === 2}>Pool</StyledTab>
+            </StyledTabList>
           </Box>
-          <Button full size="sm" onClick={() => change(Operation.SHORT)}>
-            Open Short
-          </Button>
-          <Spacer size="sm" />
-          <Button
-            full
-            disabled={!positions.loading ? false : true}
-            size="sm"
-            variant="secondary"
-            onClick={() => change(Operation.CLOSE_SHORT)}
-          >
-            Close Short
-          </Button>
           <Spacer />
-        </StyledColumn>
+          <TabPanel>
```

Comment:
For readability's sake, you may want to separate tab panel contents into different components, that's what I always end up having to do with tab views.